games-action/assault-android-cactus+/assault-android-cactus+-200507.ebuild

```diff
@@ -9,6 +9,7 @@ MY_PN="cactus"
 DESCRIPTION="Arcade style twin stick shooter set in a vivid sci-fi universe"
 HOMEPAGE="https://www.assaultandroidcactus.com/"
 SRC_URI="AssaultAndroidCactus_linux_${PV}.zip"
+
```

Comment:
I'll take it this time but I'd rather you didn't start filing PRs about adding a blank line here. There's no rule about it and I don't know why it's there so I often leave it out. I think its common presence stems from the vim ebuild template but I've never used that.

Reply:
You're right! Sorry, i just wanted to update the ebuilds accordingly to the comments, but I see that this was a bit pointless actually. Anyway, thanks, and I won't do any such useless changes again.